### PR TITLE
Update Component Dependencies

### PR DESCRIPTION
**Pull request by sareh**
**Overall change:** Update to use latest dependencies `psammead-styles@0.3.2` & `gel-foundations@0.3.0` 

Visually, there should be no changes.

`gel-foundations@0.3.0` changed the media query values to be `rem` instead of `em`. 

**Code changes:**

- `psammead-brand` & `psammead-figure` - updates `gel-foundations` from v`0.1.4` to `0.2.4` (Typography is changed from em to rem, but these components don't have visible text, so there are no visible changes)
- `psammead-brand` updates `psammead-styles` from `0.1.6` to `0.3.2` (This adds new colours and fonts, but these should not have any visual change.)
- `psammead-brand`, `psammead-copyright`, `psammead-paragraph`, `psammead-headings`, `psammead-sitewide-links` `psammead-figure` - updates `gel-foundations` from v`0.2.4` to `0.3.0` (This updates the media query values from `em` to `rem`. )

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-sitewide-links/package.json [hunk 1]
```diff
@@ -23,14 +23,14 @@
   },
   "homepage": "https://github.com/bbc/psammead/blob/latest/packages/components/psammead-sitewide-links/README.md",
   "dependencies": {
-    "@bbc/gel-foundations": "^0.2.2",
-    "@bbc/psammead-styles": "^0.3.0",
+    "@bbc/gel-foundations": "^0.2.5",
```
Comment (dr3) [inline packages/components/psammead-sitewide-links/package.json]: Can we not pull in `0.3.0` here? I know we dont want to pull `gel-foundations` into simorgh, but we should be fine pulling it in here

Comment (sareh) [reply packages/components/psammead-sitewide-links/package.json]: @dr3 Chris wanted to wait until https://github.com/bbc/psammead/issues/388 had been fixed before pulling in `gel-foundations@0.3.0`

Comment (dr3) [reply packages/components/psammead-sitewide-links/package.json]: drew [10:06 AM]
So https://github.com/bbc/psammead/issues/388 is blocking us fixing the site wide links container? which is visually broken

chrisbashton [10:06 AM]
yep :+1:

drew [10:07 AM]
I think i disagree
>The 0.3.0 of gel-foundations (use of rem over em for breakpoints)

Thats not a bug that needs to be fixed in `gel-foundations`
Our usage of it in grid needs to be fixed
We can use rems in sitewide links, as it pulls in gel-foundations on its own
We can then pull in the new sitewide links component without affecting the page grid

chrisbashton [10:10 AM]
So you disagree that 388 blocks work on the sitewide links component
Makes sense :+1:

Comment (dr3) [reply packages/components/psammead-sitewide-links/package.json]: We should be good to pull it into pssamead, as long as we dont pull it into simorgh directly

Comment (sareh) [reply packages/components/psammead-sitewide-links/package.json]: Sounds good. Will update this PR.

Comment (sareh) [reply packages/components/psammead-sitewide-links/package.json]: Changes have been made.

**Review by dr3 — APPROVED** [review]
LGTM :)
Tested using storybook, looks great, inc sitewidelinks fix
no clue why snyk isnt happy though

**Comment by ChrisBAshton** [comment]
Screen Shot 2019-03-27 at 11 38 03
Think it just had a temporary outage ^

**Review by ChrisBAshton — APPROVED** [review]
Code looks good - and thanks for adding `prettier-ignore` to all the tables.

Should do a UX review next 👍

**Comment by jamesbrumpton** [comment]
👍

**Pull request merged**